docs/source/data/data_catalog.md

```diff
@@ -143,6 +143,22 @@ test_dataset:
       encoding: "utf-8"
 ```
 
+If you want to save a file in append mode instead of overwrite you can use the `open_args_save` `mode` parameter:
+
+```yaml
+test_dataset:
+  type: ...
+  fs_args:
+    open_args_save:
+    mode: "a"
```

Comment:
Is `mode` supposed to be inside `open_args_save`?

Reply:
Ah yes, good spot. I've updated it.